modules/runner/src/integrationTest/java/org/apache/ignite/internal/runner/app/IgnitionTest.java

```diff
@@ -87,7 +85,7 @@ void testNodesStartWithBootstrapConfiguration() {
     @Test
     @Disabled("https://issues.apache.org/jira/browse/IGNITE-14709")
     void testNodeStartWithoutBootstrapConfiguration() {
-        Ignite ignite = IgnitionManager.start(null);
+        Ignite ignite = IgnitionManager.start(null, null);
```

Comment:
what we expect when we start node without bootstarping configuration and with null as name? As far as  I can understand, we'll fall with assert, looks strange